### PR TITLE
feature: add new chains + update react wallet ui

### PR DESCRIPTION
**Pull request by kasrakhosravi**
## Description:

* Adding `OKC Mainnet` and `Avalanche` (and their testnets) to list of chains
* Update wallet UI to separate `mainnet`from `testnet` chains

Screenshot 2022-08-03 at 11 06 36

## PR Checklist
- [x] :eyes: I have self-reviewed my PR, leaving comments on any changes which may trigger a discussion
- [ ] :pencil2: Added relevant tests

### CODE REVIEW

File: packages/react-wallet/src/constants/chains.ts [hunk 1]
```diff
@@ -1,6 +1,93 @@
...
+  },
+  {
+    id: 65,
+    name: 'OKC Testnet',
+    nativeCurrency: { name: 'OKT', symbol: 'OKT', decimals: 18 },
+    rpcUrls: {
+      default: 'https://exchaintestrpc.okex.org',
+    },
+    blockExplorers: {
+      etherscan: {
```
Comment (kasrakhosravi) [inline packages/react-wallet/src/constants/chains.ts:32]: @aramalipoor I couldn't fund the `etherscan blockExplorer` for `OKC` and `Avalanche`; and it was mandatory to put the value for it as pat of the chain object; so I used their own `blockExplorer` for both

Comment (aramalipoor) [reply packages/react-wallet/src/constants/chains.ts:32]: This is etherscan for Avalanche: https://snowtrace.io/

And for OKC you can put their in-house explorer `https://www.oklink.com/en/okc/` looks like their URLs are compatible with etherscan, better than nothing.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (aramalipoor) [inline .gitignore]: Usually better to put global (not specific to project) ignores to `.gitignore_global` https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

**Review by aramalipoor — APPROVED** [review]
🚀

**Comment by aramalipoor** [comment]
Can you also add Fantom and Near protocol (https://doc.aurora.dev/) as well? 😄 If yes then we can also add it to our evm-contracts to have the factories there too.

File: packages/react-wallet/src/constants/chains.ts (hunk 1 above)
Comment (kasrakhosravi) [reply packages/react-wallet/src/constants/chains.ts:32]: Yep, added both :)

**Pull request merged**